integrations/streamer_test.go

```diff
@@ -4110,7 +4110,7 @@ stream
 	}
 }
 
-func TestStream_Alert_WithReset(t *testing.T) {
+func TestStream_Alert_WithReset_0(t *testing.T) {
 	requestCount := int32(0)
```

Comment:
@nathanielc I'm not sure how to name two test functions of the same feature so I split them into `TestStream_Alert_WithReset_0` and `TestStream_Alert_WithReset_1`. What do you think?